osfoffline/views/preferences.py

```diff
@@ -36,12 +35,39 @@ def __init__(self):
 
         self.preferences_window.changeFolderButton_2.clicked.connect(self.update_sync_nodes)
         self.tree_items = []
+        self.checked_items = []
         self.setup_slots()
 
+    def get_guid_list(self):
+        guid_list = []
+        for tree_item in self.tree_items:
+            for name, id in [(node.name, node.id) for node in self.remote_top_level_nodes]:
```

Comment:
Just for future reference, try to avoid variable names that shadow builtins (like the function `id()`).